schemas/test/en/test_answer_codes.json

```diff
@@ -0,0 +1,169 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Other input fields",
+    "theme": "default",
+    "description": "A questionnaire to demo answer codes.",
+    "messages": {
+        "NUMBER_TOO_LARGE": "Number is too large",
+        "NUMBER_TOO_SMALL": "Number cannot be less than zero",
+        "INVALID_NUMBER": "Please enter an integer"
+    },
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Linear",
+        "options": {
+            "summary": {
+                "collapsible": false
+            }
+        }
+    },
+    "answer_codes": [
+        {
+            "answer_id": "mandatory-checkbox-answer",
+            "value": "None",
+            "code": "1a"
+        },
+        {
+            "answer_id": "mandatory-checkbox-answer",
+            "value": "Ham & Cheese",
+            "code": "1b"
+        },
+        {
+            "answer_id": "mandatory-checkbox-answer",
+            "value": "Ham",
+            "code": "1c"
+        },
+        {
+            "answer_id": "mandatory-checkbox-answer",
+            "value": "Pepperoni",
+            "code": "1d"
+        },
+        {
+            "answer_id": "mandatory-checkbox-answer",
+            "value": "Other",
+            "code": "1e"
+        },
+        {
+            "answer_id": "other-answer-mandatory",
+            "code": "1f"
+        },
+        {
+            "answer_id": "name-answer",
+            "code": "2"
+        },
+        {
+            "answer_id": "name-answer-2",
+            "code": "3"
+        }
+    ],
+    "sections": [
+        {
+            "id": "default-section",
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "mandatory-checkbox",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "mandatory-checkbox-answer",
+                                        "mandatory": true,
```

Comment:
I would make this voluntary so we can test, no answer codes are sent if no answer.

Reply:
Updated this 👍